packages/client/src/publication/index.ts

```diff
@@ -1,8 +1,8 @@
 export type {
-  CreateCollectBroadcastItemResultFragment,
-  CreateCommentBroadcastItemResultFragment,
-  CreateMirrorBroadcastItemResultFragment,
-  CreatePostBroadcastItemResultFragment,
+  CreateCollectTypedDataFragment,
+  CreateCommentTypedDataFragment,
+  CreateMirrorTypedDataFragment,
+  CreatePostTypedDataFragment,
```

Comment:
there is no need for keeping backward compatibility as these types are only introduced in #279